guides/micronaut-http-client/micronaut-http-client.adoc

```diff
@@ -56,12 +56,6 @@ callout:http-request[4]
 <6> GitHub encourages to explicitly request the version 3 via the `Accept` header. With `@Header`, you add the `Accept: application/vnd.github.v3+json` HTTP header to every request.
 <7> Use `retrieve` to perform an HTTP request for the given request object and convert the full HTTP response's body into the specified type. e.g. `List<GithubRelease>`.
 
-NOTE: Instead of `retrieve` we could have used `jsonStream`. You can use `jsonStream()` to stream arrays of type `application/json` or
-JSON streams of type `application/x-json-stream`. If we use `retrieve`, such as in the previous code listing, the operation will not block.
-However, it will not return until all the data has been received from the server. In the case of a JSON array that would be the whole array.
-However, if you are interested in just the first element of the array, `jsonStream` provides a better alternative since it starts streaming data from the server without needing the whole response.
-For example, `jsonStream().firstElement()` will only parse the first item in a JSON array. Hence it is more efficient.
-
```

Comment:
I removed this, as `jsonStream` is not a method of `HttpClient` and I considered it too complex for this guide to start talking about the StreamingHttpClient at this point in the docs